### ISSUE TITLE
Does not Save on version 0.7.9 of Foundry

### ISSUE THREAD

**Issue by pete284** [Handyfon/roll-of-fate#4]
You cannot save any settings on version 0.7.9 of Foundry.

errors from Console:

`foundry.js:2499 TypeError: game.settings.get(...).replaceAll is not a function
    at fate.js:384
    at Function._call (foundry.js:2496)
    at Function.call (foundry.js:2481)
    at ROFConfig._render (foundry.js:4547)
    at async ROFConfig._render (foundry.js:5157)
_call @ foundry.js:2499
$%7BsBackgroundImg%7D:1 

Failed to load resource: the server responded with a status of 404 (Not Found)

fate.js:326 Saved
foundry.js:5253 TypeError: document.getElementById(...).value.replaceAll is not a function
    at ROFConfig._updateObject (fate.js:331)
    at ROFConfig._onSubmit (foundry.js:5251)
_onSubmit @ foundry.js:5253

fate.js:326 Saved
foundry.js:5253 TypeError: document.getElementById(...).value.replaceAll is not a function
    at ROFConfig._updateObject (fate.js:331)
    at ROFConfig._onSubmit (foundry.js:5251)`

**Comment by juan-r-apolinario** [comment]
I'm having the same issue.


fate.js:326 Saved
TypeError: document.getElementById(...).value.replaceAll is not a function
    at ROFConfig._updateObject (fate.js:331)
    at ROFConfig._onSubmit (foundry.js:5251)
_onSubmit @ foundry.js:5253
async function (async)
_onSubmit @ foundry.js:5251

**Comment by Handyfon** [comment]
I can't seem to replicate this issue, do you have any modules enabled?

**Comment by pete284** [comment]
All modules disabled. Tried it in version 0.79 with DND 5e and the Simple World Building system neither one worked. Did a quick gif to show [Test](https://imgur.com/6YKUWs0)

**Comment by juan-r-apolinario** [comment]
I tested with no other modules enabled as well. Foundry up to date as your module too, using the PF2e system.

There is any specific test you would like me to do?

**Comment by Handyfon** [comment]
Did you use Roll Of Fate before? - Try reinstalling the module
Which browser do you use?
Is there an error when opening the Customizer? It doesn't seem to load correctly.

**Comment by pete284** [comment]
This is weird, yes I use Roll-of-Date in version 0.7.8 of Foundry with no issues. Other users on Discord recorded an issue (namely @KyoYagami (Juan) and @Altrix) so I installed 0.79 on a separate machine and found the same issues too.

I have re-installed Roll-of-Fate - no change Tried with Pf2e system and a new world - no change, it's really bizarre.

I have attached screenshots of the two errors when opening the configuration settings.
![roll-of-fate1](https://user-images.githubusercontent.com/55640052/104168786-7ed6d200-53f6-11eb-868a-5575051c84b5.png)
![roll-of-fate2](https://user-images.githubusercontent.com/55640052/104168799-85fde000-53f6-11eb-9189-e820135f2c39.png)

I hope this helps.
